source/jormungandr/tests/journey_common_tests.py

```diff
@@ -1589,3 +1601,211 @@ def test_night_bus_filter_parameters_with_maximum_values_3(self):
+                'timeframe_duration={timeframe_duration}&').format( _from='stop_area:sa1',
+                                                                    to='stop_area:sa3',
+                                                                    datetime='20180315T080000',
+                                                                    timeframe_duration=3600)
+        response = self.query_region(query)
+        assert 6 == len(response['journeys'])
+
+        # Time frame to catch only the first two journeys before the date time, because clockwise is active.
+        query = ('journeys?from={_from}&'
+                'to={to}&'
+                'datetime={datetime}&'
+                'datetime_represents={datetime_represents}&'
+                'timeframe_duration={timeframe_duration}&').format( _from='stop_area:sa1',
+                                                                    to='stop_area:sa3',
+                                                                    datetime='20180315T083500',
+                                                                    datetime_represents='arrival',
+                                                                    timeframe_duration=1200)
+        response = self.query_region(query)
+        assert 2 == len(response['journeys'])
+
+        assert response['journeys'][0]['departure_date_time'] == u'20180315T083000'
+        assert response['journeys'][1]['departure_date_time'] == u'20180315T082000'
+
+
+    def test_timeframe_duration_with_minimum_value(self):
+        """
+        The data contains 20 journeys (every 10 min) + 1 journey 24H after the first.
+        The first journeys is 20180315T080000.
+
+        If timeframe_duration is set to 0, the response should return 1 journey,
+        as there is simply no contraint
+        """
+        query = ('journeys?from={_from}&'
+                'to={to}&'
+                'datetime={datetime}&'
+                'timeframe_duration={timeframe_duration}&').format( _from='stop_area:sa1',
+                                                                    to='stop_area:sa3',
+                                                                    datetime='20180315T080000',
+                                                                    timeframe_duration=0)
+        response = self.query_region(query)
+        assert 1 == len(response['journeys'])
+
+    def test_timeframe_duration_and_min_nb_journeys_with_minimum_value(self):
+        """
+        The data contains 20 journeys (every 10 min) + 1 journey 24H after the first.
+        The first journeys is 20180315T080000.
+
+        If timeframe_duration and min_nb_journeys are set to 0, the response should return 1 journey,
+        as there is simply no contraint
+        """
+        query = ('journeys?from={_from}&'
+                'to={to}&'
+                'datetime={datetime}&'
+                'timeframe_duration={timeframe_duration}&').format( _from='stop_area:sa1',
```

Comment:
we should also test `min_nb_journeys=2&timeframe_duration=0` once we agree on the behaviour